wpilibj/src/main/java/edu/wpi/first/wpilibj/AnalogAccelerometer.java

```diff
@@ -30,7 +30,7 @@
    */
   private void initAccelerometer() {
     HAL.report(tResourceType.kResourceType_Accelerometer,
-                                   m_analogChannel.getChannel());
+        m_analogChannel.getChannel());
```

Comment:
http://checkstyle.sourceforge.net/config_misc.html#Indentation

See `forceStrictCondition`

Reply:
I don't actually think we want to do this